packages/lodestar/src/eth1/impl/ethers.ts

```diff
@@ -138,6 +139,41 @@ export class EthersEth1Notifier extends (EventEmitter as { new(): Eth1EventEmitt
     return Buffer.from(depositCountHex.substr(2), 'hex').readUIntLE(0, 6);
   }
 
+  public async getEth1Data(config: IBeaconConfig, state: BeaconState, currentEpoch: Epoch): Promise<Eth1Data> {
+    const [head, latestStateBlock] = await Promise.all([
+      this.getHead(),
+      this.getBlock('0x' + state.eth1Data.blockHash.toString('hex'))
+    ]);
+    const validVotes = await this.filterValidVotes(config, state.eth1DataVotes, head, latestStateBlock);
```

Comment:
can we get a type?

Reply:
it will get type automatically though

Reply:
hmm... do we have an interface defined for `vote`?

Reply:
vote is Eth1Data object.

filterValidVotes methods returns promise of Eth1Data array

Reply:
ah ok